tools/download-test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+while [[ $# -gt 1 ]]
+do
+  key="$1"
+
+  case $key in
+      -t|--tap)
+      TAP="$2"
+      shift # past argument
+      ;;
+      *) # unknown option
+      ;;
+  esac
+  shift # past argument or value
+done
+
+if [ -z ${NODE_VERSION} ]; then
+  echo "NODE_VERSION is undefined! Please declare NODE_VERSION"
+  exit 1
+fi
+# Remove old files
+[ ! -z ${TAP} ]; rm -r ${TAP} || true
+rm -r nodejs.org || true
+
+# Scrape the download server
+wget -r --no-parent --execute="robots = off" http://nodejs.org/dist/v${NODE_VERSION}/
```

Comment:
The plan is to enforce strict https on that server soon anyway, so you might be best to adapt to it now anyway.

Perhaps it might be best to go the rsync route off unencrypted:

```
rsync rsync://unencrypted.nodejs.org
```